.gitignore

```diff
@@ -36,4 +35,16 @@ yarn-error.log*
 ### VisualStudioCode Patch ###
 # Ignore all local history of files
 .history
-.ionide
\ No newline at end of file
+.ionide
+
+# Idea IDE
+# IntelliJ IDEA files
+# Ignore everything in the folder .idea except .idea/externalDependencies.xml
+**/.idea/*
```

Comment:
Are you sure that IntelliJ IDEA is a necessary or good approach to run this project?

Reply:
I improved the commentary to make it clear that this applies to all Jbrains IDEs, in this case, WebStorm for Javascript.